docs/antctl.md

```diff
@@ -571,6 +572,47 @@ $ antctl traceflow -S pod1 -D svc1 -f tcp --live-traffic -t 1m
 $ antctl traceflow -D pod1 -f tcp,tcp_dst=80 --live-traffic --dropped-only -t 10m
 ```
 
+### PacketCapture
+
+`antctl packetcapture` (or  `antctl pc`) command is used to start a `PacketCapture`
+and retrieve the captured result. After the result packet file is copied out,
+the PacketCapture will be deleted. Users can also create a PacketCapture with `kubectl`,
+but `antctl` provide a simpler way. For more information about PacketCapture,
+refer to [PacketCapture guide](packetcapture-guide.md).
+
+To start a PacketCapture, users must provide the following arguments:
+
+* `--source` (or `-S`)
+* `--destination` (or `-D`)
+* `--number` (or `-n`)
+
+Note: one of `--source` and `--destination` must be a pod.
+
+The `--flow` (or `-f`) argument can be used to specify the PacketCapture packet
+headers with the [ovs-ofctl](http://www.openvswitch.org//support/dist-docs/ovs-ofctl.8.txt)
+flow syntax(This argument works similar as Traceflow). The supported flow fields
+include: IP protocol (`icmp`, `tcp`, `udp`), source and destination ports
+(`tcp_src`, `tcp_dst`, `udp_src`, `udp_dst`).
+
+By default, the command will wait for the PacketCapture to succeed or fail, or
+timeout. The default timeout is 10 seconds, but can be changed with the
+`--timeout` (or `-t`) argument. Add the `--no-wait` flag to start a PacketCapture
+without waiting for its results. In this case, the command will not delete the
+PacketCapture resource.
+
+More examples of `antctl packetcapture`:
+
+```bash
+Start capturing packets from pod1 to pod2, both Pods are in Namespace default
```

Comment:
if you use the `bash` syntax highlighting, add the comment sign `#` before each description sentence